store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1684,6 +1684,11 @@ func (mvcc *MVCCLevelDB) MvccGetByKey(key []byte) *kvrpcpb.MvccInfo {
 	mvcc.mu.RLock()
 	defer mvcc.mu.RUnlock()
 
+	return mvcc.mvccGetByKeyNoLock(key)
+}
+
+// mvccGetByKeyNoLock implements the MVCCDebugger interface without Lock.
```

Comment:
No such function in the `MVCCDebugger` interface.

Reply:
Ping @BurtonQin

Reply:
I copy-pasted the comment above the original `MvccGetByKey()`: https://github.com/pingcap/tidb/blob/a49effd47ed9cd9af45717bff5093160b482fc09/store/mockstore/mocktikv/mvcc_leveldb.go#L1682
If there is no such function in `MVCCDebuger`, should we change the comment above MvccGetByKey()? if so, what comment should we use?

Reply:
You can remove it if it's unnecessary, I guess.

Reply:
I see. I have changed the comment above the NoLock version.